src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started leveling the ground.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped leveling the ground");

```

Comment:
The stopWork() method's output message should end with a period for consistency with the doWork() method's output.